src/back/glsl.rs

```diff
@@ -68,18 +69,202 @@ pub struct TextureMapping {
     pub sampler: Option<Handle<GlobalVariable>>,
 }
 
-const SUPPORTED_CORE_VERSIONS: &[u16] = &[450, 460];
+const SUPPORTED_CORE_VERSIONS: &[u16] = &[330, 400, 410, 420, 430, 440, 450];
```

Comment:
Could you explain more (in the PR description, or in the chat) why you think this is needed (even if it's just "just because we can")?